package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "kibana_object_format",
-  "version": "0.1.7",
+  "version": "0.1.8",
   "description": "Enables objects in arrays to be configured with a field formatter.",
   "license": "Apache 2.0",
   "homepage": "https://github.com/istresearch/kibana-object-format",
   "main": "index.js",
   "kibana": {
-    "version": "7.6.2",
+    "version": "kibana",
```

Comment:
How does this work? Is this a new "version" feature, so that we don't have to be locked to a single point release @andrewaxelrod-ist?